src/test/modelmanager_test.cpp

```diff
@@ -613,7 +613,7 @@ TEST(ModelManager, HandlingInvalidLastVersion) {
     modelInstanceUnloadGuard.reset();
     status = manager.getModelInstance(modelDirectory.name, 3, modelInstance3, modelInstanceUnloadGuard);
     modelInstanceUnloadGuard.reset();
-    ASSERT_EQ(status, ovms::StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE);
+    ASSERT_EQ(status, ovms::StatusCode::MODEL_VERSION_NOT_LOADED_YET);
```

Comment:
This status name and message indicate model is in loading phase but in fact there already was loading and it failed.

Reply:
Is this behavior expected? @atobiszei

Reply:
Model failed to load due to misconfiguration. Misconfiguration is expected to be fixed so it is reasonable to expect that model will finally be loaded.